ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss

```diff
@@ -0,0 +1,391 @@
+@import '../../../styles/variables.scss';
+
+$container-width: 1200px;
+
+.main > * {
+    width: 100%;
+}
+
+section {
+    box-sizing: border-box;
+    text-align: center;
+
+    &:nth-of-type(even) {
+        background-color: #F6F7FA;
+    }
+
+    &.introduction {
+        max-width: 1440px;
+        padding-top: 108px;
+        padding-bottom: 18px;
+        text-align: center;
+        background-image: url(/assets/images/hero.png);
+        background-repeat: no-repeat;
+        background-position: bottom;
+        background-size: contain;
+
+        .title {
+            margin-top: 0;
+            margin-bottom: 16px;
+        }
+
+        .logos {
+            img:first-child {
+                margin-right: 5rem;
+            }
+        }
+    }
+
+    &.learn-more {
+        padding: 80px 0 65px;
```

Comment:
should we use rem? same for other containers paddings

Reply:
Agreed. Added at 8e4e176298eb4955d2838fd1abe0c06da8731fdb